.github/workflows/dependencies.yml

```
@@ -51,6 +52,16 @@
             type=raw,value=latest
             type=sha
 
+      - name: Detect dependency versions
+        id: versions
+        working-directory: dependencies
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:5:23: Double quote to prevent globbing and word splitting

(shellcheck)